2.6/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -151,8 +150,12 @@ function mongo_primary_member_addr() {
 
 # mongo_remove removes the current MongoDB from the cluster
 function mongo_remove() {
-  local primary_addr
-  primary_addr="$(mongo_primary_member_addr)"
+  # Declare and assign variable in the single line to ignore possible non-zero
```

Comment:
>How this "hack" with local is better that "set +e/set -e hack"?

`set -e` **always** enables strict mode, even if it wasn't activated before. Also using `||` looks more idiomatic (at least for me).